### PR TITLE
add ltoddy's rust redis client into doc

### PR DESCRIPTION
**Pull request by ltoddy**

### CODE REVIEW

File: clients.json [hunk 1]
```diff
@@ -1085,7 +1085,15 @@
     "description": "A Redis client written with the Akka IO package introduced in Akka 2.2.",
     "authors": ["chrisdinn"]
   },
-
```
Comment (itamarhaber) [inline clients.json:1088]: Please re-add the empty line

File: clients.json [hunk 2]
```diff
@@ -1095,7 +1103,6 @@
     "active": true,
     "recommended": true
   },
-
```
Comment (itamarhaber) [inline clients.json:1098]: Please re-add the empty line

File: clients.json [hunk 3]
```diff
@@ -1085,7 +1085,15 @@
     "description": "A Redis client written with the Akka IO package introduced in Akka 2.2.",
     "authors": ["chrisdinn"]
   },
-
+  {
+    "name": "redisclient",
+    "language": "Rust",
+    "repository": "https://github.com/ltoddy/redis-rs",
+    "description": "Redis client for Rust.",
+    "authors": ["ltoddy"],
```
Comment (itamarhaber) [inline clients.json:1093]: This is supposed to have Twitter handles, not GitHub's. If you don't have a Twitter handle, just remove the authors key entirely.

**Review by itamarhaber — COMMENTED** [review]
Hello @ltoddy - thanks for this and my apologies for the lateness of the reply. Please see my comments.

**Comment by ltoddy** [comment]
@itamarhaber  please see again: https://github.com/redis/redis-doc/pull/1509

**Pull request closed without merging**